tests/mocks/selenium_chrome_driver.py

```diff
@@ -1,24 +1,26 @@
 import multiprocessing
 
 import pytest
-from chromedriver_py import binary_path
 from selenium import webdriver
 from selenium.webdriver.chrome.options import Options
 from selenium.webdriver.chrome.service import Service
+from webdriver_manager.chrome import ChromeDriverManager
 
 multiprocessing.set_start_method("fork")  # Required on macOSX
 
 
 @pytest.fixture(scope="class")
-def selenium_chrome_driver(request):
+def selenium_chrome_driver(request, live_server):
```

Comment:
If live_server is included here is [this liver_server decorator here not needed?](https://github.com/communitiesuk/funding-service-design-authenticator/blob/7395f98f775d73ba2b4f89647a528664391f1b19/tests/test_accessibility.py#L55) - just wondering if it conflicts at all?

Reply:
great spot!